### PR TITLE
Support arm64 arch

### PR DESCRIPTION
**Pull request by maboelnour**
Add `skaffold` and `k9s` binaries for arm64 arch

### CODE REVIEW

File: ohmyk8s.yaml [hunk 1]
```diff
@@ -141,13 +141,20 @@
...
       plugin_path: https://github.com/jkroepke/helm-secrets
       state: present
-
 - name: skaffold installer
   hosts: k8sdev, k8sdevlite
   tasks:
+  - name: Determine the correct package link based on architecture
+    set_fact:
+      package_link: >-
+        {%- if ansible_architecture == 'aarch64' -%}
```
Comment (hammady) [inline ohmyk8s.yaml:150]: Does this get evaluated on the controller or target machine?

File: ohmyk8s.yaml [hunk 2]
```diff
@@ -141,13 +141,20 @@
...
+        https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-arm64
+        {%- else  -%}
+        https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64
```
Comment (hammady) [inline ohmyk8s.yaml:153]: Links seem identical except in the last name segment. Can we make it DRY?

File: ohmyk8s.yaml (hunk 1 above)
Comment (maboelnour) [reply ohmyk8s.yaml:158]: It's on target, i ran the script from amd64 machine controller against aarch64 target machine :

target_arch

Used task:

```yaml
- hosts: all
  tasks:
    - name: Gather architecture information
      setup:

    - name: Display target machine architecture
      debug:
        msg: "Target machine architecture: {{ ansible_architecture }}"

File: ohmyk8s.yaml (hunk 2 above)
Comment (maboelnour) [reply ohmyk8s.yaml]: Refactored all downloadable links.

**Review by hammady — APPROVED** [review]

**Pull request merged**